src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 4;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = new Random().nextInt(FIGURE_COUNT);
+        switch (index) {
+            case 0:
+                return getDefaultFigure();
+
```

Comment:
```suggestion
            case 0:
                return getRandomCircle();
```
your program should also have a possibility to return a random circle
lets make separate private methods and create random  figures of each type
this will make your code easier to read